src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php

```diff
@@ -86,77 +94,89 @@ public function close(): bool
 
     protected function doDestroy(#[\SensitiveParameter] string $sessionId): bool
     {
-        $this->getCollection()->deleteOne([
-            $this->options['id_field'] => $sessionId,
-        ]);
+        $write = new BulkWrite();
+        $write->delete(
+            [$this->options['id_field'] => $sessionId],
+            ['limit' => 1]
+        );
 
-        return true;
+        return $this->manager->executeBulkWrite($this->namespace, $write)->isAcknowledged();
```

Comment:
`isAcknowledged()` will always return true unless you set the `writeConcern` option to 0 (either as a bulk write option or as a connection option). If you're looking to return `true` if the operation succeeded, you can safely leave the previous return statement in place as the operation will throw an exception in case of an error.

Note that a false value for `isAcknowledged()` doesn't mean that the operation failed - it merely means that the driver has no way of knowing whether the operation succeeded or not.